public/signupOrLogin.html

```diff
@@ -42,13 +43,15 @@ <h2>Sign up or log in</h4>
         }
       });
       var style = {
-        fontFamily: '"Helvetica New", Helvetica, sans-serif',
+        fontFamily: 'Helvetica',
```

Comment:
Let's use this full font family name.
```suggestion
        fontFamily: '"Helvetica New", Helvetica, sans-serif',
```